paintBrush.js

```diff
@@ -62,7 +69,33 @@ function stopPainting() {
     painting = false;
 }
 
+function eraseTrue() {
+    eraseMode = true;
+}
+
+function eraseFalse() {
+    eraseMode = false;
+}
+
 function startPainting() {
     painting = true;
 }
 
+
+function colourPicker(e){
+    colourMode = e.currentTarget.name
+}
+
+function clickShow(e){
+    mode = e.currentTarget.name
+    buttons.forEach(function(button){
+        if (button.name === mode) {
+            button.classList.add('clicked')
+        }
+        else if (button.name !== mode) {
+            button.classList.remove('clicked')
+        }
```

Comment:
There is a `classlast.toggle()` method that you might be able to use here to remove complexity